docs/template_plugin/tests/functional/shared_tests_instances/behavior/ie_class.cpp

```diff
@@ -6,7 +6,11 @@
 #include <string>
 #include <vector>
 
-#include "ie_class.hpp"
+#include "behavior/ie_class.hpp"
```

Comment:
Can we rename to `core_integration.hpp`?

Reply:
yeah

Reply:
Done